giraffe/package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "build": "rm -rf dist && NODE_ENV=production webpack --config webpack.config.js",
     "lint": "eslint '{src,../stories/src}/**/*.{ts,tsx}'",
-    "prettier": "prettier --config ../.prettierrc.json --check '{src,../stories/src}/**/*.{ts,tsx}'",
-    "prettier:fix": "prettier --config ../.prettierrc.json --write '{src,../stories/src}/**/*.{ts,tsx}'",
+    "prettier": "pretty-quick --config ../.prettierrc.json --check '{src,../stories/src}/**/*.{ts,tsx}'",
+    "prettier:fix": "pretty-quick --config ../.prettierrc.json --write '{src,../stories/src}/**/*.{ts,tsx}'",
```

Comment:
I would like to preserve the original scripts as well. The originals can have new names. Something like `prettier:all` and `prettier:all:fix`

Reply:
This would be the last issue and then this PR will be good to go. Thanks.

Reply:
ps:  none of the other projects preserve the original scripts.  so you can add them back if you like. (clockface and ui)